src/cli.ts

```diff
@@ -36,6 +36,11 @@ yargs.command("$0 <grammar> [output_file]", "Build parser from grammar",
                 default: false,
                 desc: "Use numeric enums for AST kinds",
             },
+            "union-enums": {
```

Comment:
I think this flag is too low-level, I think it should be called "erasable-syntax" so that it represents why you might use it, rather than it's effect

Reply:
Just for the record, I personally also prefer string constant unions over enums in general, it's also e.g. what the [acorn parser uses](https://github.com/acornjs/acorn/blob/master/acorn/src/acorn.d.ts). So I can just use a string without having to import some enum, e.g.

```ts
function processNode(node) {
  switch (node.kind) {
     case "sum":
       return node.left. + node.right;
     case "product":
       return node.left. * node.right;
}
```

That it also works with `--erasableSyntaxOnly` is a bonus -- I would probably enable the flag anyways.  But perhaps that's just me, I can rename it if you want.